templates/sitemap/sitemap.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <sitemap>
+    <loc>{{ base_url }}/sitemap-links.xml</loc>
+  </sitemap>
+  <sitemap>
+    <loc>{{ base_url }}/sitemap-operators.xml</loc>
+  </sitemap>
+  <sitemap>
+    <loc>{{ base_url }}/docs/sitemap.xml</loc>
```

Comment:
can you remove this one, we are not indexing yet the docs pages

Reply:
Sure!